README.md

```diff
@@ -1,336 +1,248 @@
+```
+
+Learn more about Composite Resources in the [Crossplane
 Docs](https://crossplane.io/docs/v1.9/concepts/composition.html).
 
-## Pre-Requisite & Optional Tools
-
-Install the following command line tools:
-
-* `up cli`
-
-  There are multiple ways to [install up](https://cloud.upbound.io/docs/cli/#install-script), including Homebrew and Linux packages.
-
-  ```console
-  curl -sL https://cli.upbound.io | sh
+## Quickstart
 
-  ```
+### Pre-Requisites
 
-## Platform Ops/SRE: Run your own internal cloud platform
+Before we can install the reference platform we want to install the `up` CLI.
+This is a utility that makes following this quickstart guide easier. Everything
+described here can also be done in a declarative approach which we highly
+recommend for any production type use-case.
+<!-- TODO enhance this guide: Getting ready for Gitops -->
 
-The Universal Crossplane (UXP) can be provisioned to any Kubernetes cluster.
-
-The AWS Reference platform will extend Kubernetes API with your own platform API
-abstractions.
-
-#### Installing UXP on a Kubernetes Cluster
-
-The other option is installing UXP into a Kubernetes cluster you manage using `up`, which
-is the official CLI for interacting with Upbound Cloud and Universal Crossplane (UXP).
+To install `up` run this install script:
+```console
+curl -sL https://cli.upbound.io | sh
+```
+See [up docs](https://docs.upbound.io/cli/) for more install options.
 
-Ensure that your kubectl context is pointing to the correct cluster:
+For installing the platform we need a running Crossplane control plane. We are
+using [Universal Crossplane (UXP)
+](https://github.com/upbound/universal-crossplane). Ensure that your kubectl
+context is pointing to the correct Kubernetes cluster or for example create a
+[kind](https://kind.sigs.k8s.io) cluster:
 
 ```console
-kubectl config current-context
+kind create cluster --image kindest/node:v1.25.0
 ```
 
-Install UXP into the `upbound-system` namespace:
+Finally install UXP into the `upbound-system` namespace:
 
 ```console
-up login
 up uxp install
 ```
 
-Validate the install using the following command:
+You can validate the install by inspecting all installed components: 
 
 ```console
 kubectl get all -n upbound-system
 ```
 
-#### Install the Platform Configuration
+### Install the AWS Reference Platform
 
-Now that your kubectl context is configured to connect to a UXP Control Plane,
-we can install this reference platform as a Crossplane package.
+Now you can install this reference platform. It's packaged as a [Crossplane
+configuration package](https://crossplane.io/docs/v1.9/concepts/packages.html)
+so there is a single command to install this package: 
 
 ```console
-# Check the latest version available in https://marketplace.upbound.io/configurations/upbound/platform-ref-aws/
-kubectl apply -f examples/configuration.yaml
-kubectl get pkg
+up ctp configuration install xpkg.upbound.io/upbound/platform-ref-aws:v0.4.0
```

Comment:
```suggestion
up ctp configuration install xpkg.upbound.io/upbound/platform-ref-aws:v0.4.1
```
Let's use the version with the fixed provider-helm dependency

Reply:
In addition, I still believe that we should keep `examples/configuration.yaml` and reference it as 'optional GitOps-compatible way of installation'